metadata/metadata.go

```diff
@@ -244,12 +217,14 @@ func updateEtag(resp *http.Response) bool {
 	return etag != oldEtag
 }
 
-func watchMetadata(ctx context.Context) (*metadata, error) {
-	return getMetadata(ctx, true)
+// Watch runs a longpool on metadata server.
```

Comment:
nit: `long poll` here and all other usages?

Reply:
done.